client/src/Components/Categories/Categories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./category.scss";
+
+const Categories = ({ categoryList }) => {
+
+  const orderedCategories = categoryList
+    .filter((item) => item.order != -1)
+    .sort((item1, item2) => {
+      return item1.order - item2.order;
+    });
+    
+  return (
+    <main>
+      {orderedCategories.map((item, key) => {
+        return (
+          <section key={key} className="category-item">
+            <article>
+              <h3>{item.name}</h3>
+              <p>{item.description}</p>
+            </article>
+            <figure>
+              <img
+                src={item.imageUrl}
+                alt="image loading"
+                height="100"
```

Comment:
is there a alt text already available in "item". If not, is this a valid alt text anyway?